src/sagas/wallet.js

```diff
@@ -349,6 +361,37 @@ export function* listenForFeatureFlags(featureFlags) {
   }
 }
 
+export function* listenForPushNotificationFeatureFlag(featureFlags) {
+  const channel = eventChannel((emitter) => {
+    const listener = (state) => emitter(state);
+    featureFlags.on(FeatureFlagEvents.PUSH_NOTIFICATION_ENABLED, (state) => {
+      emitter(state);
+    });
+
+    // Cleanup when the channel is closed
+    return () => {
+      featureFlags.removeListener(FeatureFlagEvents.PUSH_NOTIFICATION_ENABLED, listener);
+      featureFlags.offUpdatePushNotification();
+    };
+  });
+
+  try {
+    while (true) {
+      const newUsePushNotification = yield take(channel);
+      const oldUsePushNotification = yield select((state) => state.pushNotification.use);
```

Comment:
Is it working? I don't see this `state.pushNotification.use` state in redux.

Reply:
[0ac170](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/240/commits/0ac170f2fd8f832853d5f5e92212c4fee269c81c)

Reply:
Yes, I changed from `use` to `available` as you suggested. It remained working because it only is used to determine if the state needs to be updated. With this error every time the state will be updated.